src/crypto/verthash.cpp

```diff
@@ -120,12 +120,14 @@ void Verthash::Hash(const char* input, char* output)
     const uint32_t mdiv = ((datfile_sz - HASH_OUT_SIZE)/BYTE_ALIGNMENT) + 1;
     
     if(!datFileInRam) {
+        size_t read_len = -1;
```

Comment:
Any reason not to scope this variable to the same line as the fread?

Reply:
I was (wrongly) assuming this causes overhead - but it looks like compilers optimize this away. Changed